tools/ogs/ExtractBoundary.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="extractboundary" name="OGS extractBoundary" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>
+        Extracts a boundary from a mesh.
+    </description>
+    <macros>
+        <import>macros.xml</import>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <expand macro="ogs_requirement"/>
+    </requirements>
+        
+        <expand macro="creator"/>
+    <command detect_errors="exit_code"><![CDATA[
+    ln -s '$input_mesh' input.vtu &&
+        ExtractBoundary -i input.vtu -o outputExtractBoundary.vtu $ascii_output
+    ]]></command>
+    <inputs>
+        <param name="input_mesh" type="data" format="vtkxml" label="Input Mesh File" help="The 3D mesh file (in VTU or MSH format) from which the boundary will be extracted." />
+        <param name="ascii_output" type="boolean" optional="true" value="false" truevalue="--ascii-output" falsevalue="" label="ASCII Output" help="If selected, the output mesh will be saved in ASCII format instead of binary." />
+    </inputs>
+    <outputs>
+        <data name="output_mesh" format="vtkxml" label="Extracted Boundary Mesh" from_work_dir="outputExtractBoundary.vtu"/>
```

Comment:
If there is just a single output I would prefer to go with the default `label`, i.e. `${tool.name} on ${on_string}`.